Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "yagna"
-version = "0.3.0"
+version = "0.3.3-alpha.4"
```

Comment:
```suggestion
version = "0.3.3-alpha.5"
```

Reply:
@tworec I'd suggest `0.4.0` since ExeUnit uses a completely different communication api with the vm runtime

Reply:
is it back compatible? I need some sync with you on that

Reply:
but this seems legit afai understand

Reply:
@tworec on a second thought, `ya-exe-unit` and `ya-provider` are versioned separately, so I'm leaving this change to other reviewers